platform/jvm/replay/BUILD.bazel

```diff
@@ -15,11 +15,13 @@ bitdrift_kt_android_library(
     visibility = ["//visibility:public"],
     deps = [
         "//platform/jvm/common:lib",
-        artifact("androidx.appcompat:appcompat"),
         artifact("androidx.compose.ui:ui"),
-        artifact("org.jetbrains.kotlin:kotlin-stdlib"),
         artifact("com.squareup.okhttp3:okhttp"),
```

Comment:
(if this works) you could also move this one and remove it from capture since it's repeated there